shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -74,9 +78,10 @@ private static String fromFlutterAssets(String filePath) {
     private static String sFlutterAssetsDir = DEFAULT_FLUTTER_ASSETS_DIR;
 
     private static boolean sInitialized = false;
-    private static ResourceExtractor sResourceExtractor;
+    private static ResourceExtractor sResourceExtractor = null;
     private static boolean sIsPrecompiledAsBlobs;
     private static boolean sIsPrecompiledAsSharedLibrary;
+    private static boolean aotAbiSplit;
```

Comment:
Change to `sAotAbiSplit` for consistency

Or maybe something like `sIsPrecompiledAsBlobLibraries`?